packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -737,6 +737,10 @@ Examples:
                 (x) => `"${x}"`
             ).join(', ')})`
         )
+        .option(
+            '--templateVersion <version>',
+            `The version of the template to be generated. This defaults to 'latest'. NOTE: This option only applies to templates being downloaded from NPM.`
+        )
```

Comment:
With this change, you can get this help message when running `node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js -h`.

```suggestion
        .option(
            '--templateVersion <version>',
            `The version of the template to be generated. NOTE: This option only applies to templates being downloaded from NPM.`,
            'latest'
        )
```

![NeoVim Terminal 2023-05-29 at 11 16 02](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/847300/441fc0d5-2221-4cfc-9639-a2723cc1eea3)

Reply:
Thanks.. this is a good solution and follows the current patten to how we are using args.